lib/src/main/kotlin/starknet/data/types/transactions.kt

```diff
@@ -103,18 +103,18 @@ data class DeclareTransaction(
     }
 }
 
-data class DeployTransaction(
-    val contractDefinition: CompiledContract,
-    val contractAddressSalt: Felt,
-    val constructorCalldata: Calldata,
-    val nonce: Felt?
-) : Transaction() {
-    override val type = TransactionType.DEPLOY
-
-    override fun getHash(): Felt {
-        TODO("Not yet implemented")
-    }
-}
+// data class DeployTransaction(
+//    val contractDefinition: CompiledContract,
+//    val contractAddressSalt: Felt,
+//    val constructorCalldata: Calldata,
+//    val nonce: Felt?
+// ) : Transaction() {
+//    override val type = TransactionType.DEPLOY
+//
+//    override fun getHash(): Felt {
+//        TODO("Not yet implemented")
+//    }
+// }
```

Comment:
👁️

Reply:
Well, I didn't want to remove it here, as there is an issue for getting rid of data class duplication (#43) and I think it's better to get rid of these classes there.

Reply:
But you commented it out here anyway, so there is a change. Why do we need a commented version of it here?